subprojects/ide/src/main/java/org/gradle/plugins/ide/eclipse/model/EclipseModel.java

```diff
@@ -217,6 +223,31 @@ public void jdt(Action<? super EclipseJdt> action) {
         action.execute(getJdt());
     }
 
+    /**
+     * The list of tasks to execute before the Eclipse synchronization starts.
+     *
+     *
+     *
+     * @return the tasks names
+     * @since 5.4
+     */
+    @Incubating
+    public List<String> getSynchronizationTasks() {
```

Comment:
❌ These shouldn't be Strings, but `Task`s.

Reply:
Fixed. In order to parse tasks from String format, I had to inject the current project.